src/GridItem.ts

```diff
@@ -45,6 +45,9 @@ class GridItem {
   public isUpdate = false;
   public hasTransition = false;
   public transitionDuration = "";
+  public isRestoreOrgCSSText = true;
+  public isRequestUpdate = false;
```

Comment:
Looks like there's already `isUpdate` beside `isRequestUpdate`.
Wouldn't it be confusing if these two are mixed together?

Reply:
@WoodNeck 

The two properties have different roles.

So would it be okay to change isUpdate to isUpdating?

Reply:
What about using words that can better describe itself?
At first look, my question for `isRequestUpdate` was
- What is the update for? What's the difference between `Update` and `RequestUpdate`?
- Who is requesting the update? Is it something requested from outside of the class?

If the `isRequestUpdate` is for the re-rendering items, what about `shouldRerender` or `isReupdate` or something?

Reply:
`Update`: Whether or not it will be updated upon request.
`RequestUpdate`: Whether the item needs to be updated again

Inside the grid class, the size of the item changes, so the update is requested again.

I will fix it with `shouldReupdate`.